### DISCUSSION TITLE
Invalid Refresh Token

### DISCUSSION THREAD

**Discussion by RaySolo** [dani-garcia/vaultwarden#4625] [Q&A]
### Subject of the issue: 
Upgraded the macOS client and the vaultwarden server to the current versions endet up with all clients unable to sync with server or add new entries with error 'Invalid Refresh Token'. 

### Deployment environment

* vaultwarden version: 1.30.5

* Install method: Docker

* Clients used: 
iOS, macOS, Wondows 11
* Reverse proxy and version: 
nginx reverse proxy behind firewall
* MySQL/MariaDB or PostgreSQL version: 
MySQL
* Other relevant details:
everthing worked fine before. Installing an older client doesn't fix the problem
### Steps to reproduce

before upgrading I logged out with the client apps and started the vaultwarden container with a new image. The volume docker/vault/data is linked to /data inside the container. After a few seconds the container showed up healthy and I tried to connect via macOS client. After providing the credentials the client window appeared empty, even though the top right corner telling me my eMail address and my server domain name. on my Mac with an older version of the client I still can see my data, but adding or modifying any entry brings up the error message.
### Expected behaviour

After updating the server i expected to get all my data when logging into it. 
### Actual behaviour

Starting the client and typing in the master password shows the startscreen of the app, but with none data.

### Troubleshooting data

[Abb_ 2024-06-04 um 15.44.16.pdf](https://github.com/user-attachments/files/15566396/Abb_.2024-06-04.um.15.44.16.pdf)

**Comment by BlackDex** [comment]
Either linked to #4503 so ensure you have the latest client version installed. 

Or something in between is causing issues like WAF, ModSecurity, CloudFlare etc... 

Also check the logs of Vaultwarden during this attempt, the screenshot doesn't provide anything useful.
